### ISSUE TITLE
BUG: Cannot connect using SSH key file

### ISSUE THREAD

**Issue by KBulmer** [dbgate/dbgate#173]
Hello,

I'm getting the following error when trying to connect to a MySQL database:
```
Error: Cannot parse privateKey: Encrypted private OpenSSH key detected, but no passphrase given
    at e.exports (Z:\dbgate-windows-latest\resources\app.asar\packages\api\dist\bundle.js:15:8492)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async process.<anonymous> (Z:\dbgate-windows-latest\resources\app.asar\packages\api\dist\bundle.js:191:427282)

Error JSON: {}

Connection: {
  "server": "***",
  "engine": "mysql@dbgate-plugin-mysql",
  "sshMode": "keyFile",
  "sshPort": "***",
  "sshKeyfile": "***",
  "useSshTunnel": true,
  "sshHost": "***",
  "sshLogin": "***",
  "user": "***",
  "password": "***",
  "displayName": "***",
  "sshKeyfilePassword": "***",
  "useSsl": false,
  "_id": "***",
  "status": "***"
}

Platform: win32
```

The passphrase is correct and works in other applications.

Windows 10

Version: 4.3.0 (portable)
Build date: 2021-09-20

**Comment by janproch** [comment]
Thanks for reporting. Is fixed in newest BETA, 4.3.1-beta.2

**Comment by janproch** [comment]
Released in 4.3.1
